rllib/algorithms/algorithm.py

```diff
@@ -311,40 +308,50 @@ def from_state(state: Dict) -> "Algorithm":
     @PublicAPI
     def __init__(
         self,
-        config: Optional[Union[PartialAlgorithmConfigDict, AlgorithmConfig]] = None,
-        env: Optional[Union[str, EnvType]] = None,
+        config: Union[AlgorithmConfig, PartialAlgorithmConfigDict],
```

Comment:
Deprecated passing `env` into Algo's c'tor. Should no longer be used and now causes an error.